src/apps/src/Eryph-zero/OVSPackage.cs

```diff
@@ -52,7 +48,8 @@ public static string UnpackAndProvide(bool canInstallDriver, string? relativePac
         foreach (var ovsFilesDir in ovsRootDir.GetDirectories("run_*"))
         {
             log.Verbose("Checking path '{ovsFileDir}' for ovs files", ovsFilesDir.FullName);
-            var hasOvs = ovsFilesDir.GetFiles("ovs-vswitchd.exe", SearchOption.AllDirectories).Any();
+            var hasOvs = ovsFilesDir.GetFiles("ovs-vswitchd.exe", SearchOption.AllDirectories).Any()
+                && ovsFilesDir.GetFiles("dbo_ovse.inf", SearchOption.AllDirectories).Any();
```

Comment:
has the structure changed? currently a run dir don't contains the driver, it is stored one folder above.

Reply:
Yes. The old logic copied the driver from the run dir to the separate driver directory. I thought it might be better to keep the driver inside the run dir. This way the version check for the driver can always check that the installed version matches the version in the run dir and hence the OVS version which will actually be used.